tiatoolbox/utils/visualization.py

```diff
@@ -1011,7 +925,7 @@ def render_annotations(
             for i, (key, box) in enumerate(bounding_boxes.items()):
                 area = (box[0] - box[2]) * (box[1] - box[3])
                 if area > min_area or i % decimate == 0:
-                    ann = store.__getitem__(key, True)
+                    ann = store[key]
```

Comment:
I think @measty mentioned this was intentional to use `__getitem__`

Reply:
Which is no longer needed because of this PR